users/admin.py

```diff
@@ -34,7 +35,7 @@ def link(self, obj):
     return link
 
 
-class JustfixUserAdmin(airtable.sync.SyncUserOnSaveMixin, UserAdmin):
+class JustfixUserAdmin(airtable.sync.SyncUserOnSaveMixin, UserAdmin, MapModelAdmin):
```

Comment:
This ensures the user admin includes the CSS and JS needed to render the maps on the client-side.